src/helpers/visibilityClassnames.ts

```diff
@@ -0,0 +1,11 @@
+import { display } from 'classnames/tailwind'
+
+export const displayFromXs = display('hidden', 'xs:flex')
```

Comment:
Maybe we can make those function more uniform? Like:
```TS
export const displayFrom = (from: 'xs' | 'md' | 'lg') => display('hidden', `${from}:flex`)
```
Let's modify `displayTo` like this as well
You can even do `displayFromTo` 
++ you can choose which thing to display, like:
```TS
export const displayFrom = (from: From, box: 'flex' | 'block') => display('hidden', `${from}:${box}`)
```

Reply:
Take types from tailwind, of course